opencog/cython/opencog/Cast.cc

```diff
@@ -8,3 +8,4 @@ using namespace opencog;
 Handle atom_from_the_void(long p) { return *((Handle*) p); }
 long void_from_candle(const Handle& hp) { return (long) (&hp); }
 long void_from_cptr(Handle* hp) { return (long) (hp); }
+long avoid_from_cptr(Handle* hp) { return (long) (hp->operator->()); }
```

Comment:
Its the pointer to the actual atom. Same value as `hp->get()`

Recall that Handles are just smart pointers, they only point to the actual atom.  The raw pointer is quasi-hidden, so that users won't user Atom*  but you can still get Atom* by saying wither hp->get or hp->operator->() -- either one works.